src/anshuKJun20/SumOfNumberFromString.java

```diff
@@ -0,0 +1,22 @@
+package anshuKJun20;
+
+public class SumOfNumberFromString {
+
+	void findSumOfNumber(String input) {
+		int sum = 0;
+		for (int index = 0; index < input.length(); index++) {
+			if (Character.isDigit(input.charAt(index)) == true) { // Interger.Parseint() takes String and return integer
+				int num = Integer.parseInt(input.valueOf(input.charAt(index))); // valueOf() take character and returns string
```

Comment:
You can use below method to convert char to int
Character.getNumericValue(numberString.charAt(index));